ansible_ai_connect/ai/api/model_pipelines/wca/pipelines_base.py

```diff
@@ -389,10 +394,132 @@ class WCABasePlaybookGenerationPipeline(
     def __init__(self, inference_url):
         super().__init__(inference_url=inference_url)
 
+    def invoke(self, params: PlaybookGenerationParameters) -> PlaybookGenerationResponse:
```

Comment:
I just moved this function from "SaaS" to the common base class for both "SaaS" and "on-prem".